tools/clang/test/CodeGenDXIL/validation/val-SampleCmpBiasGrad-shader-model-vs68.ll

```diff
@@ -0,0 +1,147 @@
+; RUN: not %dxv %s 2>&1 | FileCheck %s
```

Comment:
Won't filecheck's exit code be the code that lit uses to determine whether the test passed? why is 'not' needed here?

Reply:
lit doesn't like command fail. not will make the command pass.
taef allows command to fail, so we don't need not for taef.